### PR TITLE
Added volume controls, fixed panics from issue #3

### PR DESCRIPTION
**Pull request by IguanasInPyjamas**

### CODE REVIEW

**Comment by IguanasInPyjamas** [comment]
Can't set reviewer for some reason. As requested, volume controls exist now @MattAG

File: src/uri.rs [hunk 1]
```diff
@@ -6,8 +6,10 @@ pub fn setup_pipeline(path:&std::string::String) -> gstreamer::Element {
     //Relatively naive, needs to be future proofed
     let mut filepath = std::string::String::from("");
     if path.contains("https") || path.contains("http"){
-        path.replace(" ","%20");
-        filepath = path.to_string();
+
+        filepath = path.replace(" ","%20");
+        filepath = filepath.to_string();
```
Comment (MattAG) [inline src/uri.rs:11]: This could just be 1 line, to_string after replace.

**Review by MattAG — COMMENTED** [review]
Good stuff.

File: src/uri.rs (hunk 1 above)
Comment (IguanasInPyjamas) [reply src/uri.rs]: Done

**Pull request merged**